### PR TITLE
Display full menu + dietary icons for each menu item

### PR DESCRIPTION
**Pull request by plorry**
Summary: 

- new file with constants for icon emoji values
- functions to map zerocater API `label` values to corresponding icons
- expansion of functionality populating templates; feeds in full menu item list
- templates expanded to display this new information
- tests updated to match new function output

### CODE REVIEW

File: lib/lita/handlers/icons.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ICONS = {
```
Comment (plorry) [inline lib/lita/handlers/icons.rb:1]: A note that the Zerocater API actually includes a few more labels than the ones below; I have curated the ones that I think cover the most common dietary concerns, and avoid confusion.

**Comment by esigler** [comment]
Awesome!  Thanks for the PR @plorry !  Will merge shortly.

**Comment by plorry** [comment]
@esigler hoping to get this puppy flying soon - lemme know if you're good for merge / deploy!

**Comment by esigler** [comment]
I'm curious if end-users will want a "volume knob", where some may want the additional info and some not so much.  Let's find out!

**Pull request merged**